### PR TITLE
gff.awn.gff_awngi v1.0

### PR DESCRIPTION
**Pull request by dyacob**
Introduction of an Awngi lexicon, extracted from a newspaper corpus.

This lexicon will pair with a new keyboard for Awngi & Xamtanga languages which will be submitted in the coming days. Perhaps I'm getting ahead of myself but thought to start the review process early for the lexicon.

### CODE REVIEW

**Comment by keyman-server** [comment]
This pull request is from an external repo and will not automatically be built. The build must still be passed before it can be merged. Ask one of the team members to make a manual build of this PR.

**Comment by DavidLRowe** [comment]
HISTORY.md has version 1.0 as expected for a new model, but several other files use version 1.0.1. Not a big deal.

You are welcome to remove the version number and the copyright date (or the entire copyright line) from README.md. That will mean fewer files to change (just to change date or version number) when revising the model.

**Review by DavidLRowe — APPROVED** [review]
I'll approve this "as is", but wait to merge it until tomorrow. If you want to make changes to version numbers, your new commit will dismiss the review and I'll review it again. Otherwise I'll merge this PR.

**Pull request merged**
